docs/jwst/resample/arguments.rst

```diff
@@ -109,8 +109,27 @@ image.
 ``--blendheaders`` (bool, default=True)
     Blend metadata from all input images into the resampled output image.
 
-
 ``--in_memory`` (boolean, default=True)
   Specifies whether or not to load and create all images that are used during
   processing into memory. If ``False``, input files are loaded from disk when
   needed and all intermediate files are stored on disk, rather than in memory.
+
+``--enable_ctx`` (boolean, default=True)
+  Specifies whether or not to compute and store the context array (`con`) in the datamodel,
+  which is used to track which input images contributed to each pixel in the
+  output image. Setting this to ``False`` helps reduce memory usage for very large
+  mosaics.
+
+``--enable_err`` (boolean, default=True)
+  Specifies whether or not to compute and store the error array in the output model.
+  Setting this to ``False`` helps reduce memory usage and output file size for very
+  large mosaics, but the  `var`, `err`, `var_flat`, `var_rnoise`, and `var_poisson`
+  arrays will not be computed or reported.
+  If set to ``False``, the `enable_var` flag is ignored.
+
+``--enable_var`` (boolean, default=True)
+  Specifies whether or not to store the variance arrays, namely
+  `var`, `var_flat`, `var_rnoise`, and `var_poisson`, in the output model.
```

Comment:
No 'var' array anyway, I think this comment should be just 'var_flat', 'var_rnoise', and 'var_poisson'